app/apptesting/txfees.go

```diff
@@ -0,0 +1,85 @@
+package apptesting
```

Comment:
The methods in the file are extracted from previously existing txfees helpers because they need to be used in other module tests, and I wanted to prevent code duplication.